packages/usa-in-page-navigation/src/index.js

```diff
@@ -0,0 +1,216 @@
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const { CLICK } = require("../../uswds-core/src/js/events");
+
+const CURRENT_CLASS = `${PREFIX}-current`;
+const IN_PAGE_NAV_TITLE = "On this page";
+const IN_PAGE_NAV_CLASS = `${PREFIX}-in-page-nav`;
+const IN_PAGE_NAV_ANCHOR_CLASS = `${PREFIX}-anchor`;
+const IN_PAGE_NAV_LIST_CLASS = `${IN_PAGE_NAV_CLASS}__list`;
+const IN_PAGE_NAV_ITEM_CLASS = `${IN_PAGE_NAV_CLASS}__item`;
+const IN_PAGE_NAV_LINK_CLASS = `${IN_PAGE_NAV_CLASS}__link`;
+const IN_PAGE_NAV_TITLE_CLASS = `${IN_PAGE_NAV_CLASS}__heading`;
+const SUB_ITEM_CLASS = `${IN_PAGE_NAV_ITEM_CLASS}--sub-item`;
+const MAIN_ELEMENT = "main";
+
+// Set Intersection Observer options
+const IO_ROOT = null;
+const IO_ROOT_MARGIN = "0px 0px -85% 0px";
+const IO_THRESHOLD = [0];
+
+const options = {
+  root: IO_ROOT,
+  rootMargin: IO_ROOT_MARGIN,
+  threshold: IO_THRESHOLD,
+};
+
+/**
+ * Set the active link state for the currently observed section
+ *
+ * @param {HTMLElement} el An element within the in-page nav component
+ */
+const setActive = (el) => {
+  const allLinks = document.querySelectorAll(`.${IN_PAGE_NAV_LINK_CLASS}`);
+  el.map((i) => {
+    if (i.isIntersecting === true && i.intersectionRatio >= 1) {
+      allLinks.forEach((link) => link.classList.remove(CURRENT_CLASS));
+      document
+        .querySelector(`a[href="#${i.target.id}"]`)
+        .classList.add(CURRENT_CLASS);
+      return true;
+    }
+    return false;
+  });
+};
+
+/**
+ * Return a node list of section headings
+ *
+ * @return {HTMLElement[]} - An array of DOM nodes
+ */
+const getSectionHeadings = () => {
+  const sectionHeadings = document.querySelectorAll(
+    `${MAIN_ELEMENT} h2, ${MAIN_ELEMENT} h3`
+  );
+  return sectionHeadings;
+};
+
+/**
+ * Return a node list of section anchor tags
+ *
+ * @return {HTMLElement[]} - An array of DOM nodes
+ */
+const getSectionAnchors = () => {
+  const sectionAnchors = document.querySelectorAll(`.${IN_PAGE_NAV_ANCHOR_CLASS}`);
+  return sectionAnchors;
+};
+
+/**
+ * Return a section id/anchor hash without the number sign
+ *
+ * @return {String} - Id value with the number sign removed
+ */
+const getSectionId = (value) => {
+  let id;
+
+  // Check if value is an event or element and get the cleaned up id
+  if (value && value.nodeType === 1) {
+    id = value.getAttribute("href").replace("#", "");
+  } else {
+    id = value.target.hash.replace("#", "");
+  }
+
+  return id;
+};
+
+/**
+ * Scroll smoothly to a section based on the passed in element
+ *
+ * @param {HTMLElement} - Id value with the number sign removed
+ */
+const handleScrollToSection = (el) => {
+  window.scroll({
+    behavior: "smooth",
+    top: el.offsetTop,
+    block: "start",
+  });
+};
+
+const createInPageNav = (inPageNavEl) => {
+  const sectionHeadings = getSectionHeadings();
+  const inPageNav = document.createElement("nav");
+  inPageNav.setAttribute("role", "navigation");
```

Comment:
This line is probably safe to remove. 
`<nav>` has an implicit role of "navigation", so I believe there is no need to make it explicit. (Reference: [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav))

Reply:
Addressed this by changing role to your recommended aria-label... Commented on this below as well, thanks!